src/System.Net.Security/tests/UnitTests/System/PinnableBufferCacheTest.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    public class PinnableBufferCacheTest
+    {
+        [Fact]
+        public void PinnableBufferCache_AllocateBuffer_Ok()
+        {
+            var cacheName = "Test";
+            var numberOfElements = 5;
+            var p = new PinnableBufferCache(cacheName, numberOfElements);
+
+            var a = p.AllocateBuffer();
```

Comment:
Nit: here and elsewhere, please use `var` only if the type is named on the right-hand side, e.g. casts and ctors.